File: giovana-bot/src/models/machine.js
'use strict';

const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const schema = new Schema({

    Name: {
        type: String
    },
    Status: {
        type: String
    },
    DtUpdate: {
        type: Date
    }
});

module.exports = mongoose.model('Machine', schema);

File: giovana-service/ManagerMachine/Models/Machine.cs
using MongoDB.Bson;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ManagerMachine.Models
{
    public class Machine
    {
        public ObjectId Id { get; set; }
        public string Name { get; set; }
        public string Status { get; set; }
        public DateTime DtUpdate { get; set; }
    }
}
